base/sysinfo.jl

```diff
@@ -377,7 +384,19 @@ See documentation in [Handling Operating System Variation](@ref).
 """
 isapple(os::Symbol) = (os === :Apple || os === :Darwin)
 
-for f in (:isunix, :islinux, :isbsd, :isapple, :iswindows, :isfreebsd, :isopenbsd, :isnetbsd, :isdragonfly)
+"""
+    Sys.isjsvm([os])
+
+Predicate for testing if Julia is running in a JavaScript VM (JSVM), including e.g.
+WebAssembly in a browser or JavaScript embedding.
+WASM in a Browser/JS embedding)
```

Comment:
```suggestion
```

Reply:
Sorry, I guess this didn't get picked up by the previous suggestion. I'm not sure how to make them multi-line...

Reply:
It's not a Browser or Javascript embedding though, it's a JS embedding in the browser.

Reply:
Okay, how about delete this line as in the suggestion and amend the line above it to be
```
WebAssembly in a JavaScript embedding in the browser.
```
?

Reply:
I'd say `including e.g. a WebAssembly JavaScript embedding in the browser`, because it's a WebAssembly embedding in JavaScript.

Reply:
Okay, sounds good to me.